src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -272,6 +272,14 @@
     /** Subdirectory string where we store {@link FlowNode}s */
     private String storageDir = null;
 
+    static {
+        // Prevents the sole Timeout thread from being created where it will get a Groovy classloader and cause memory leaks
+        try (Timeout tim = Timeout.limit(1, TimeUnit.SECONDS)){
```

Comment:
This is better fixed in `Timeout`.

Reply:
cf. https://github.com/jenkinsci/workflow-support-plugin/pull/53

Reply:
It's coming, just takes a little bit for the deployment chain to catch up.